### DISCUSSION TITLE
Record Plugin for windows desktop app

### DISCUSSION THREAD

**Discussion by TheBurntWaffl3** [llfbandit/record#236] [Q&A]
Hey there! 
I'm attempting to use the plugin within a windows desktop app created with visual studio code. However, every time I attempt to start the recording, the app simply closes, with the debug console stating "Lost connection to device." I have a simple app with a _toggleaudio function that's called when a button is pressed:

void _toggleAudio() async {

    final record = AudioRecorder();
    if (await record.hasPermission()) {
    await record.start(const RecordConfig(), path: './recordings');
    }
  }
Appreciate any help or insight!

**Comment by llfbandit** [comment]
The error message seems weird.
First, have a look to the sample provided with the package.
Using path_provider and naming correctly the destination file should help.
